html_decorators.py

```diff
@@ -1,22 +1,23 @@
 def div(func):
-    # You have to code here!
-    pass
+    def add_div(text):
+        return '<div>' + func(text) + '</div>'
```

Comment:
Usa mejor [Formatted String Literals](https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals) o [format() ](https://docs.python.org/3/tutorial/inputoutput.html#the-string-format-method) ya que no sabes si `func(text)` tiene implamentado `__add__`